apps/extension/background-script.ts

```diff
@@ -38,26 +38,21 @@ scripting.registerContentScripts([
   }
 ]);
 
-runtime.onConnect.addListener(port => {
+runtime.onConnect.addListener(async port => {
   // check for time expired and max-view no opened then extension need to lock
-  const savedSessionTimeExpired = Date.now() > lastUserActivityTimestamp + APP_LOCK_PERIOD;
 
   if (isFullpagePort(port)) {
     isFullpageOpen = true;
   }
 
-  if (savedSessionTimeExpired && !isFullpageOpen) {
-    passwordHash = INITIAL_PASSWORD_HASH;
-  }
-
   // listen when UI is closed
   if (port.name === 'klaytn_wallet_ui') {
     port.onDisconnect.addListener(port => {
       if (isFullpagePort(port)) {
         isFullpageOpen = false;
       }
 
-      return (lastUserActivityTimestamp = Date.now());
+      return putToStorage({ [LAST_USER_ACTIVITY_TIMESTAMP_KEY]: Date.now() });
```

Comment:
take a look at this line, `setSessionPaswordHash` and `setLockTimePeriod`.  
  
1. you are using different approaches to the same functionality.
2. these functions have identical code - that means, something wrong with it